r/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN echo ".libPaths(c('$RENV_LIB', .libPaths()))" >> $(R RHOME)/etc/Rprofile.sit
 
 # install renv to install required R packages
 RUN R -q -e "install.packages('remotes', repos = c(CRAN = 'https://cloud.r-project.org'))" && \
-    R -q -e "remotes::install_github('rstudio/renv@0.16.0')" && \
+    R -q -e "remotes::install_github('rstudio/renv@v1.0.7')" && \
```

Comment:
same suggestion, changing to PPM cran repo to speedup build.

Reply:
single package/dependencies installation probably not worthwhile